internal/errs/errs.go

```diff
@@ -125,3 +125,19 @@ func Matches(err error, target interface{}) bool {
 	}
 	return false
 }
+
+func JoinWrappedErrors(err error) string {
```

Comment:
This functionality already exists: http://github.com/ActiveState/cli/blob/f054c636e6eab88e5fb7dd38a3d693aa8eda6222/internal/errs/errs.go#L75-L75

Reply:
look at that...